src/python/pants/engine/target.py

```diff
@@ -2466,10 +2473,35 @@ def unevaluated_transitive_excludes(self) -> UnparsedAddressInputs:
         )
 
 
+# If this predicate returns false, then the target's field's deps will be ignored.
+ShouldTraverseDepsPredicate = Callable[[Target, Field], bool]
+
+
+# noinspection PyUnusedLocal
+def should_traverse_deps_default_predicate(tgt: Target, fld: Field) -> bool:
```

Comment:
I think these ought to be spelled out (same below). The only time I'm in favor of shortening is if it collides with a keyword.